### DISCUSSION TITLE
btn-active without btn-group

### DISCUSSION THREAD

**Discussion by st11x** [saadeghi/daisyui#2029] [Q&A] [answered]
Hi,

btn-group is going to be removed, However, btn-active does not work if you don't have the btn-group parent (with the new join).

How do you make the button active without resorted to changing the color yourself? 

Matt

**Comment by saadeghi** [comment, marked as answer]
`btn-active` and `btn-primary` were both applying the same color. `btn-active` was just some kind of alias.  
You can use `btn-primary` to get the same result.

Let me know if you have any questions.

**Reply by st11x** [reply]
Thanks for the clarification.
